src/common/components/ListItemAction/ListItemAction.jsx

```diff
@@ -6,6 +6,8 @@ import { Button } from 'antd';
 import IconText from '../IconText';
 import './ListItemAction.scss';
 
+const CLASS_NAME = '__ListItemAction__';
```

Comment:
maybe something related to a button?

Reply:
this is how we name root node class for a component though.

Reply:
in this case is just for the button, no?

Reply:
it's the root node for this component though.